src/deps.ts

```diff
@@ -1,21 +1,17 @@
-export { assert, assertEquals, assertRejects, assertThrows } from 'https://deno.land/std@0.141.0/testing/asserts.ts'
-export * as base64 from 'https://deno.land/std@0.141.0/encoding/base64.ts'
-export * as flags from 'https://deno.land/std@0.141.0/flags/mod.ts'
-export * as colors from "https://deno.land/std@0.141.0/fmt/colors.ts"
-export * as fs from 'https://deno.land/std@0.141.0/fs/mod.ts'
-export * as path from 'https://deno.land/std@0.141.0/path/mod.ts'
-export * as streams from "https://deno.land/std@0.141.0/streams/mod.ts"
-export { default as tweetnacl } from 'https://esm.sh/tweetnacl@1.0.3?pin=v120'
-export { base58btc } from 'https://esm.sh/multiformats@11.0.2/bases/base58?pin=v120'
-export { type Multibase } from 'https://esm.sh/multiformats@11.0.2?pin=v120'
-export { default as blake } from 'https://esm.sh/@multiformats/blake2@1.0.13?pin=v120'
-export { CID } from 'https://esm.sh/multiformats@11.0.2/cid?pin=v120'
-export { miniexec as sh } from "https://deno.land/x/miniexec@1.0.0/mod.ts"
-export * as esbuild from "https://deno.land/x/esbuild@v0.14.47/mod.js"
-export * as sqlite from "https://deno.land/x/sqlite@v3.7.1/mod.ts"
-export { type DB as SQLiteDB, type PreparedQuery } from "https://deno.land/x/sqlite@v3.7.1/mod.ts"
-// Other third-party shell libs:
-// https://deno.land/x/shell_tag
-// https://deno.land/x/violet
-// https://deno.land/x/dzx
-// https://deno.land/x/miniexec
+export { assert, assertEquals, assertRejects, assertThrows } from 'jsr:@std/assert';
+export * as base64 from 'jsr:@std/encoding/base64';
+export * as flags from 'jsr:@std/flags/';
+export * as colors from 'jsr:@std/fmt/colors';
+export * as fs from 'jsr:@std/fs/';
+export * as path from 'jsr:@std/path/';
+export * as streams from 'jsr:@std/streams/';
+export * as util from 'jsr:@std/io';
+export { copy, readAll } from 'jsr:@std/io';
```

Comment:
The previous import lines pinned a specific version, while the new imports don't seem to do that. Can you please use a specific version?